instrumentation/tomcat-7.0/javaagent/tomcat-7.0-javaagent.gradle

```diff
@@ -0,0 +1,22 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.tomcat.embed"
+    module = "tomcat-embed-core"
+    // Tomcat 10 is about servlet 5.0
+    // 7.0.4 added Request.isAsync, which is needed
+    versions = "[7.0.4, 10)"
+  }
+}
+
+dependencies {
+  library group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '7.0.4'
+  implementation project(':instrumentation:servlet:servlet-3.0:javaagent')
+  implementation("org.slf4j:slf4j-api:1.7.30")
```

Comment:
I wonder, why no other instrumentation defines this dependency?

Reply:
removed